Userland/Libraries/LibWeb/DOM/ProcessingInstruction.cpp

```diff
@@ -13,6 +16,7 @@ ProcessingInstruction::ProcessingInstruction(Document& document, DeprecatedStrin
     : CharacterData(document, NodeType::PROCESSING_INSTRUCTION_NODE, data)
     , m_target(target)
 {
+    set_prototype(&Bindings::ensure_web_prototype<Bindings::ProcessingInstructionPrototype>(document.realm(), "ProcessingInstruction"));
```

Comment:
I don't think this should need the ensure API and the bindings header included here -- or in character data tbh. cached_web_prototype should do the trick assuming ProcessingNode is marked with [Exposed=Window].